satellite/console/consoleweb/consoleql/consoleerrors.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package consoleql
+
+import (
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/satellite/console"
+)
+
+// Error messages
+const (
+	internalErrDetailedMsg = "It looks like we had a problem on our end. Please try again"
+)
+
+// errConsoleInternalDetailed describes detailed error message for internal error
+var errConsoleInternalDetailed = errs.New(internalErrDetailedMsg)
+
+// HandleError returns detailed error if such error handles
+func HandleError(err error) error {
+	switch {
+	case console.ErrConsoleInternal.Has(err):
+		return errConsoleInternalDetailed
+	default:
+		return err
```

Comment:
add default error msg

Reply:
We don't yet have default error. 
Currently now, our default error is ``Internal Error``. 
This change is not in scope of this task.